Explain idle connection handling (#1443)
moq-wg/moq-transport: draft-ietf-moq-transport.md

exhaustion attacks by a peer that does not send expected data within
an expected time.  Each implementation is expected to set its own timeouts.

### Idle Connection Handling

> A MoQT session may become idle when a publisher has no active subscribers, or

Reviewer: Did you mean subscriptions or something else?

```suggestion
A MoQT session may become idle when a publisher has no active subscriptions, or
```

when a subscriber has no active subscriptions. In these cases, the underlying
transport connection (e.g., QUIC) can close due to idle timeout if no data is
exchanged.

Implementations that need to maintain idle sessions have several options:

* Use transport-layer keep-alive mechanisms, such as QUIC PING frames, to
  prevent idle timeout closure.

> * Send periodic MoQT control messages (TODO: Grease once it's defined?).

Author: ```suggestion
* Send periodic MoQT control messages, for example REQUEST_UPDATE with no
  modified Message Parameters.
```

exhaustion attacks by a peer that does not send expected data within
an expected time.  Each implementation is expected to set its own timeouts.

### Idle Connection Handling

A MoQT session may become idle when a publisher has no active subscribers, or
when a subscriber has no active subscriptions. In these cases, the underlying
transport connection (e.g., QUIC) can close due to idle timeout if no data is

> exchanged.

Author: ```suggestion
The transport connection (e.g., QUIC) underlying a MOQT session can close due to
idle timeout if no data is exchanged, either because there are no established
subscriptions or the established subscriptions are not publishing Objects
frequently.  This includes publisher sessions that have issued a
PUBLISH_NAMESPACE and are waiting for subscribers.
```
